docs/integrations/data-ingestion/clickpipes/mysql/schema-changes.md

```diff
@@ -0,0 +1,15 @@
+---
+title: 'Schema Changes Propagation Support'
+slug: /integrations/clickpipes/mysql/schema-changes
+description: 'Page describing schema change types detectable by ClickPipes in the source tables'
+---
+
+ClickPipes for MySQL can detect schema changes in the source tables and, in some cases, automatically propagate the changes to the destination tables. The way each DDL operation is handled is documented below:
+
+[//]: # "TODO Extend this page with behavior on rename, data type changes, and truncate + guidance on how to handle incompatible schema changes."
```

Comment:
we don't currently propagate any of these. to alter columns they should add new column with new name & drop old column